packages/netlify/src/index.ts

```diff
@@ -432,9 +444,7 @@ export default function netlifyIntegration(
 				rootDir = config.root;
 				_config = config;
 
-				if (config.image?.domains?.length || config.image?.remotePatterns?.length) {
-					await writeNetlifyFrameworkConfig(config, logger);
-				}
+				await writeNetlifyFrameworkConfig(config, logger);
```

Comment:
Is it safe to remove the if condition?

Reply:
Yeah, it was only there because previously it was only needed for images. As we're now using it for headers we need to write it every time